### PR TITLE
re-added opalite file in different branch

### PR DESCRIPTION
**Pull request by bleehu**
new file:   01_04_Species_Opaleites.txt

### CODE REVIEW

**Comment by bleehu** [comment]
This one Opaleite file is holding back the whole rest of the species re-work. I recommend pulling it off to the side and working on in as it's own feature.

**Review by Reveraine — APPROVED** [review]
Good! Ship It!

**Comment by bleehu** [comment]
Not quite ready yet...

**Comment by bleehu** [comment]
Currently blocked by #781 , #297, #782  and a couple of others.

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 1]
```diff
@@ -2,7 +2,21 @@
 Opaleites
 ==============================
 
-  Opaleites are an organic amphibious species that 
+    Opaleites are an organic amphibious species that evolved on a planet that 
+was mostly covered by marshes, seas and oceans. Their unique resperatory system
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt:6]: *respiratory

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 2]
```diff
@@ -2,7 +2,21 @@
...
+lets them breathe under water as well as on land. An Opaleite's eyes are a 
+little better at seeing underwater then they are out of it.
+
+    Play as an Opaleite if you ...
+
+*  want to stick close to your team as if they were your schoolmates.
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt:12]: I'd move the "if you" down to this line. Makes for consistent parallel structure.

File: CharacterCreation/01_04_Species_Opaleites.txt [hunk 3]
```diff
@@ -14,6 +28,12 @@ underwater sounds. Keeping the second set closed does hurt an Opaleite's speed
 while swimming. They are also used to drain the lungs when making the switch 
 from water to air.
 
+    Opaleites have two legs and two arms. Each leg has a foot with five webbed
+toes. Each arm has four webbed fingers and an aposable thumb. The arms also have
```
Comment (rosebrallier) [inline CharacterCreation/01_04_Species_Opaleites.txt:32]: *opposable

**Review by rosebrallier — COMMENTED** [review]
Liking the update!

**Comment by Turtlelord26** [comment]
@bleehu No longer blocked, right?

**Pull request merged**